src/utils/git-commit.js

```diff
@@ -44,6 +44,7 @@ async function get(path) {
   return git.config({fs, dir: '.', path});
 }
 async function getGlobalConfig() {
-  const file = await readFile(`${process.env.HOME}/.gitconfig`).catch(() => '');
+  const home = ((process.env.HOME: any): string);
```

Comment:
It looks like [`process.env` is typed as](https://github.com/facebook/flow/blob/5fcf1b69f56dc2d2699125c58380c2ade1a3f73d/lib/node.js#L2166):

{code}
env: { [key: string] : ?string };
{code}

So if you wanted to clean this up without having to unsafely case to `any`, you could instead add a refinement checking that `process.env.HOME` is defined.